task-13/src/main/java/com/epam/winter_java_lab/task_13/controller/UserController.java

```diff
@@ -0,0 +1,61 @@
+package com.epam.winter_java_lab.task_13.controller;
+
+import com.epam.winter_java_lab.task_13.domain.Role;
+import com.epam.winter_java_lab.task_13.domain.User;
+import com.epam.winter_java_lab.task_13.dto.UserDto;
+import com.epam.winter_java_lab.task_13.service.Impl.UserService;
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.domain.Page;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.Map;
+
+@RestController
+@RequestMapping("/user")
+@Api(value = "user resources")
+public class UserController {
+
+    private UserService userService;
+
+    @Autowired
+    public UserController(UserService userService) {
+        this.userService = userService;
+    }
+
+    @PreAuthorize("hasAuthority('ADMIN')")
```

Comment:
drop PreAuthorize annotation. In the future we will choose another security approach